lib/bemhtml/runtime/index.js

```diff
@@ -389,6 +392,21 @@ BEMHTML.prototype.runOne = function runOne(json) {
   return res;
 };
 
+BEMHTML.prototype.tryRun = function tryRun(context, ent) {
+  try {
+    return ent.run(context);
+  } catch (e) {
+    console.error('BEMXJST ERROR: cannot render ' +
+      [
+        'block ' + context.block,
+        'elem ' + context.elem,
+        'mods ' + JSON.stringify(context.mods),
+        'elemMods ' + JSON.stringify(context.elemMods)
+      ].join(', '), e);
```

Comment:
@St-Lukas e.stack is useless:


93:trycatch miripiruni$ node stand.js
BEMXJST ERROR: cannot render block page, elem undefined, mods {}, elemMods {} TypeError: Cannot set property 'test' of undefined
   at ContextChild.<anonymous> (evalmachine.<anonymous>:1863:19)
   at Match.tryCatch (evalmachine.<anonymous>:1240:15)
   at Match.exec (evalmachine.<anonymous>:1290:16)
   at Entity.defaultBody (evalmachine.<anonymous>:266:26)
   at Entity.run (evalmachine.<anonymous>:251:15)
   at BEMHTML.tryRun (evalmachine.<anonymous>:681:16)
   at BEMHTML.runOne (evalmachine.<anonymous>:663:10)
   at BEMHTML._run (evalmachine.<anonymous>:531:16)
   at BEMHTML.renderContent (evalmachine.<anonymous>:970:18)
   at BEMHTML.renderClose (evalmachine.<anonymous>:831:19)